license.go

```diff
@@ -1,23 +1,56 @@
 package main
 
 import (
-	"fmt"
 	"io"
 	"os"
+	"sort"
 	"path/filepath"
 	"text/template"
 )
 
-//go:generate go-bindata -o templates.go .templates/
+var licenses = map[string]struct {
```

Comment:
The list of licenses is now a map of short name to a struct of the long name and the template file.